bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -64,7 +64,7 @@ private static int getDefaultDataChannelCount() {
     // 10 Channels seemed to work well on a 4 CPU machine, and this seems to scale well for higher
     // CPU machines. Use no more than 250 Channels by default.
```

Comment:
Our internal performance tests showed that we should be adding fewer requests on the same channel.  I doubled the channels, and significantly reduced the outstanding RPCs.  I think that this deserves a separate PR, so I'll undo these changes.